tests/tck/features/match/SeekById.intVid.feature

```diff
@@ -244,13 +244,15 @@ Feature: Match seek by id
       RETURN v.name AS Name
       """
     Then a SemanticError should be raised at runtime:
+
+  Scenario: with arithmetic
     When executing query:
       """
       MATCH (v)
       WHERE (id(v) + 1) == hash('James Harden')
       RETURN v.name AS Name
       """
-    Then a SemanticError should be raised at runtime:
+    Then the execution should be successful
```

Comment:
should add the expected results

Reply:
Since we don't know the exact value of `hash('James Harden')-1`, there may or may not be a matching result.